portfolio/src/main/webapp/gallery.js

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const galleryImages = [
+    {
+        name: "cern.jpg",
+        caption: "My family at CERN"
+    },
+    {
+        name: "deer.jpg",
+        caption: "A deer looking through a window at my house"
+    },
+    {
+        name: "kungfu.gif",
+        caption: "My twin brother and I doing a kung fu form."
+    },
+    {
+        name: "mountains.jpg",
+        caption: "A picture of me at Mount Pilatus."
+    },
+    {
+        name: "dollar.jpg",
+        caption: "Me holding a Zimbabwean dollar I won in economics class."
+    }
+];
+
+shuffle(galleryImages);
+
+let HTMLImage;
+let HTMLCaption;
```

Comment:
Same for HTMLCaption.
s/HTMLCaption/htmlCaption.

Reply:
Fixed.